physics/smoke_dust/rrfs_smoke_wrapper.meta

```diff
@@ -715,6 +730,13 @@
   type = real
   kind = kind_phys
   intent = inout
+[fire_type_out]
+  standard_name = fire_type_out
```

Comment:
Can you change the standard name to `fire_type`?

Reply:
@grantfirl Sure, we will change the standard name to fire_type. Thanks.